src/autopas/cells/ParticleCell.h

```diff
@@ -128,9 +142,50 @@ class ParticleCell {
   [[nodiscard]] virtual std::array<double, 3> getCellLength() const = 0;
 
   /**
-   * Lock object for exclusive access to this cell.
+   * Get the type of particles contained in this cell. Possible values:
+   * dummy: this cell is empty
+   * owned: this cell can ONLY contain owned particles
+   * halo: this cell can ONLY contain halo particles
+   * ownedOrHalo: this cell can contain owned or halo particles
+   * @return type of particles inside this cell
+   */
+  const OwnershipState getPossibleParticleOwnerships() { return _ownershipState; }
```

Comment:
```suggestion
  OwnershipState getPossibleParticleOwnerships() const { return _ownershipState; }
```
Copied `const` return values have no effect. However, The function itself should be `const`.

Reply:
done in [06385c2](https://github.com/AutoPas/AutoPas/pull/782/commits/06385c264cc416ebc407bb395db2e8d1a70c1cd0)